be/src/runtime/load_channel_mgr.h

```diff
@@ -76,10 +77,17 @@ class LoadChannelMgr {
     std::mutex _lock;
     // load id -> load channel
     std::unordered_map<UniqueId, std::shared_ptr<LoadChannel>> _load_channels;
+    std::shared_ptr<LoadChannel> _reduce_memory_channel = nullptr;
     Cache* _last_success_channel = nullptr;
 
     // check the total load channel mem consumption of this Backend
     std::shared_ptr<MemTrackerLimiter> _mem_tracker;
+    int64_t _load_process_soft_limit = -1;
+
+    // If hard limit reached, one thread will trigger load channel flush,
+    // other threads should wait on the condition variable.
+    bool _hard_limit_reached = false;
```

Comment:
We should just use memtracker as indicator of limit reaching.  Here, the variable indicates one thead is flushing memtable due to reached hard limit, so maybe we should name the variable as '_is_flushing_channel'.

Reply:
It seems that _reduce_memory_channel is enough?

Reply:
`_reduce_memory_channel` is used to indicate the soft limit triggered, to avoid multiple thread flushing due to soft limit.
`_hard_limit_reached` indicates that all thread should wait here until we released some memory